bedrock/settings/base.py

```diff
@@ -31,19 +31,40 @@ def path(*args):
 
 DEBUG = config('DEBUG', cast=bool, default=False)
 
-# Production uses MySQL, but Sqlite should be sufficient for local development.
-# Our CI server tests against MySQL.
-DATABASES = config(
-    'DATABASES',
-    cast=json.loads,
-    default=json.dumps(
-        {'default': config('DATABASE_URL',
-                           cast=dj_database_url.parse,
-                           default='sqlite:///bedrock.db')}))
 
-if DATABASES['default']['ENGINE'].endswith('psycopg2'):
-    # let the DB handle connection killing
-    DATABASES['default']['CONN_MAX_AGE'] = None
+class NonDefaultRouter(object):
+    def __init__(self, dbname):
+        self.dbname = dbname
+
+    def _return_dbname(self, *args, **kwargs):
+        return self.dbname
+
+    def _return_true(self, *args, **kwargs):
+        return True
+
+    db_for_read = _return_dbname
+    db_for_write = _return_dbname
+    allow_relation = _return_true
+    allow_migrate = _return_true
```

Comment:
A shorter suggestion:

```python
class NonDefaultRouter(object):
  def __init__(self, dbname):
    self.dbname = dbname

  db_for_read = db_for_write = lambda self, *a, **kw: self.dbname
  allow_relation = allow_migrate = lambda *a, **kw: True 
```

If you prefer the more verbose version I have no objections.